src/panels/developer-tools/service/developer-tools-service.js

```diff
@@ -118,6 +118,11 @@ class HaPanelDevService extends PolymerElement {
           <mwc-button on-click="_callService" raised disabled="[[!validJSON]]"
             >Call Service</mwc-button
           >
+          <template is="dom-if" if="[[_attributes.length]]">
+            <mwc-button on-click="_fillExampleData" raised
+              >Fill Example Data</mwc-button
```

Comment:
Since whitespace doesn't really matter here, let's keep the `>` and `</` on new lines.